octoprint_authldap/templates/authldap_settings.jinja2

```diff
@@ -0,0 +1,79 @@
+<form class="form-horizontal">
+    <h3>LDAP configuration</h3>
+    <div class="control-group">
+        <h4>General</h4>
+        <label class="control-label">{{ _('LDAP URI') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_uri"
+                   placeholder="ldaps://ldap.server.com">
+        </div>
+
+        <label class="control-label">{{ _('Search base pattern') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_search_base"
+                   placeholder="CN=Users,DC=example,DC=com">
+        </div>
+
+        <label class="control-label">{{ _('LDAP Filter') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.ldap_query"
+                   placeholder="(&(objectclass=user)(memberof=<GROUPNAME>)(samaccountname={uid}))">
+        </div>
+
+        <label class="control-label">{{ _('Default Roles:') }}</label>
+        <div class="controls">
+            <input type="text" class="input-block-level" data-bind="value: settings.plugins.authldap.roles"
+                   placeholder="Comma separated list of roles: user,admin">
```

Comment:
Self explanatory.
This will give the newly created users specific roles. Currently Octoprint only differs between user and administrator rights